packages/forms-web-app/src/controllers/householder-planning/listed-building-householder.js

```diff
@@ -0,0 +1,58 @@
+const logger = require('../../lib/logger');
+const { createOrUpdateAppeal } = require('../../lib/appeals-api-wrapper');
+
+const { VIEW } = require('../../lib/views');
+
+const listedBuilding = VIEW.HOUSEHOLDER_PLANNING.LISTED_BUILDING;
+const backLink = `/${VIEW.FULL_APPEAL.TYPE_OF_PLANNING_APPLICATION}`;
```

Comment:
This would not take the user to Type of planning application page

Reply:
Edited.